cli/utils/response.go

```diff
@@ -1,4 +1,4 @@
-package cli
+package utils
```

Comment:
I will rename this to `client` for now - we can fold response.go into http.go at a later point.